_docs/_latest/develop/dev-setup.md

```diff
@@ -185,7 +185,7 @@ following (use the values from your terminal, not "`/SOME/PATH/TO`"):
 
 Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
 `forseti-security/configs/forseti_conf.yaml`. Refer to 
-["Configuring Forseti"]({% link _docs/latest/configure/configuring-forseti.md %}) 
```

Comment:
Not on this line, but the Title and filename can be improved/shortened.

Reply:
This is part of develop, will update that in the next PR when I review develop section.